icon-typings-builder/build.js

```diff
@@ -0,0 +1,93 @@
+#! /usr/bin/env node
+/**
+ * Based on https://github.com/callemall/material-ui/tree/master/packages/icon-builder
+ */
+
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const rimraf = require('rimraf');
+const Mustache = require('mustache');
+const glob = require('glob');
+const mkdirp = require('mkdirp');
+const rrs = require('recursive-readdir-sync');
+
+const SVG_ICON_RELATIVE_REQUIRE = '../../SvgIcon';
+const OUTPUT_DIR = path.resolve(__dirname, '../svg-icons');
+const SVG_DIR = path.resolve(__dirname, './node_modules/material-design-icons/');
+
+function main() {
+  rimraf.sync(OUTPUT_DIR); // Clean old files
```

Comment:
Maybe we should simply remove this line and configure a "clean" task in package.json